src/_guides/language/language-tour.md

```diff
@@ -720,6 +718,64 @@ information about lists, see [Generics](#generics) and
 [Collections](/guides/libraries/library-tour#collections).
 
 
+### Sets
```

Comment:
We should add something about how to create a set literal for an empty set (that needs disambiguation with the map literal, so needs a type on either the left or right side, see: https://github.com/dart-lang/language/blob/master/accepted/future-releases/set-literals/feature-specification.md#examples)

Reply:
Done. PTAL.